src/Utilities/ErrorHandling/AbortWithErrorMessage.cpp

```diff
@@ -91,7 +92,7 @@ template <bool ShowTrace>
   // NOLINTNEXTLINE(cppcoreguidelines-pro-type-vararg)
   CkError("%s", os.str().c_str());
   breakpoint();
-  sys::abort("");
+  throw std::runtime_error(os.str());
```

Comment:
Please update comment above that mentions "abort".

Question: Since uncaught exceptions print a traceback by default (they do, right?), should we move the stack trace printing that's done in this function to where we catch and handle the exception, e.g., to `initiate_shutdown` (or somewhere else) where we call `std::abort`?

Reply:
stack traces are weird with exceptions and you'll only get where it was handled, not where it was thrown (or in general this depends on how the exception is handled). We should keep the current mechanism to guarantee a full trace in all cases.